langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -123,6 +157,12 @@ public isolated function min(int n, int... ns) returns int = @java:Method {
 # The first character may be `+` or `-`.
 # This is the inverse of function ``value:toString`` applied to an `int`.
 #
+# ```ballerina
+# int:fromString("76") ⇒ 76
+# 
+# int:fromString("abc") ⇒ error
```

Comment:
According to the discussions happen in the decimal and float it is better to add something like
`int:fromString("76f")`.
wdyt?

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/17cd4d61b91f614ad1cf6d330607b3c3da18da54 as per the suggestion at: https://github.com/ballerina-platform/ballerina-lang/pull/39109#discussion_r1051923249